src/EditorFeatures/Test/Diagnostics/DiagnosticProviderTestUtilities.cs

```diff
@@ -6,6 +6,7 @@
 using System.Threading;
 using Microsoft.CodeAnalysis.CSharp.Diagnostics;
 using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Diagnostics.EngineV1;
```

Comment:
Never mind - figured that this is because something in this file is using type that moved... :)